.github/workflows/iroha2.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         with:
           context: .
           push: true
-          tags: hyperledger/iroha2:2.0.0-pre-rc.2
+          tags: hyperledger/iroha2:current
```

Comment:
Distinguish from `latest`?

Reply:
`latest` sounds more idiomatic to me

Reply:
Latest means latest, as in the one that we are working on. 
Current means, the currently recommended version. 
Perhaps `stable`? But then we have `LTS`

Reply:
I think latest should refer to latest published release. What we're working on should be tagged with `-dev` or alike

Reply:
I thought this was the convention. If, I'm wrong feel free to override me

Reply:
`latest` tag semantics are overloaded so it should be avoided. I would prefer if you used `dev`

Reply:
Ah, I see. This is not for the latest development container, rather for the latest release.

Reply:
ok, so why the switch from the semantic versioning? If it's the latest release it should have a proper version assigned